src/components/status-image/status-image.jsx

```diff
@@ -1,8 +1,16 @@
 import './status-image.css';
+import standing from '../../assets/standing-19@2x.png';
+import home from '../../assets/Home.svg';
+import campus from '../../assets/Wireframe.svg';
 
 function StatusImage(props) {
     return(
-        <div className="StatusImage">StatusImage</div>
+        <div className="StatusImage" id={props.available === false ? 'opacity40' : undefined}>
```

Comment:
no id! you can also calculate the value "opacity40" before the return. I would also change "opacity40" by "busy" because it is more semantic.

Also add a tooltip (hover the component) that shows the status and place.

````js
    const {available, onCampus} = props;
    const isBusy = !available ? 'busy' : '';`
    const tooltip = `${available ? 'available': 'busy'}, ${onCampus ? 'on campus': 'home'}`;
````

```suggestion
        <div className={'StatusImage ' + isBusy} title={tooltip}}>
```